Kudu.Core/Deployment/WellKnownEnvironmentVariables.cs

```diff
@@ -36,5 +36,6 @@ internal static class WellKnownEnvironmentVariables
         public const string SiteBitness = "SITE_BITNESS";
 
         public const string GoWebConfigTemplate = "GO_WEB_CONFIG_TEMPLATE";
+        public const string SelectLatestVersionCommandKey = "SELECT_LATEST_VERSION_CMD";
```

Comment:
Yeah, as David pointed out, this is probably about selecting the latest version of Go, in which case the script and code members should be named appropriately.